### ISSUE TITLE
simplify encumbrances

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#8058]
Help us understand your request (check below):

- [X] new feature or otherwise a good idea to make Arctos even better

**Describe what you're trying to do** 

Ref https://github.com/ArctosDB/new-collections/issues/759#issuecomment-2316258273

Arctos currently has two ways to restrict access to coordinates.

1. https://arctos.database.museum/info/ctDocumentation.cfm?table=ctencumbrance_action#mask_coordinates is super-kludgy (no control over data) and probably not very secure (all the pieces are still available and there are a LOT of pathways to access Arctos data).
2. https://arctos.database.museum/info/ctDocumentation.cfm?table=ctlocality_attribute_type#locality_access allows users to publish or restrict ***exactly*** what of the locality stack they wish, whatever that might be, is secure, and I don't think has or will come to have any backdoors (it acts directly on the restricted data, not on one of the many things which might use such data from several joins away).

Can we get rid of the encumbrance? I would of course migrate any existing data to use the locality attribute.

**Comment by dustymc** [comment]
Adding to AWG agenda: Encumbrances carry a cost which has become difficult to pay, this seems like an easy/obvious way to both lower cost and provide a better experience for all users, curatorial and researcher.

**Comment by mkoo** [comment]
From 6 Feb 2025 issues meeting discussion:

We had a demo and discussion about the locality access function and once we had an example in front of us, we came to the decision that encumbrance type of mask coordinates should be removed/ replaced in favor of the locality attribute option.

Concerns are:
-  making sure or requiring that locality access attribute always has metadata of determiner and date ( can that be part of the save function to add in the user and today's date?)
- migration pathway for current mask coordinate encumbrances. (maybe allow legacy ones to stand?)
- Lastly, could be seen as a positive but locality attributes do not expire like encumbrances

**Comment by dustymc** [comment]
Discussed with @mkoo go dev for combo.
